core/comp_ui.py

```diff
@@ -44,12 +45,14 @@ def _PromptLen(prompt_str):
     elif c == '\x02':
       escaped = False
     elif not escaped:
-      display_str += c
+      display_str = display_str + c
   last_line = display_str.split('\n')[-1]
   try:
     width = libc.wcswidth(last_line)
   # en_US.UTF-8 locale missing, just return the number of bytes
-  except (SystemError, UnicodeError):
+  except SystemError:
```

Comment:
Reading the code in `pyext/libc.c` it seems it never throws SystemError?  And our C++ doesn't

So I guess delete this, it was dead code ?